tests/distribution/pom.xml

```diff
@@ -384,6 +396,14 @@
                                     <version>${carbon.kernel.version}</version>
                                 </feature>
 
+                                <feature>
+                                    <id>org.wso2.carbon.cluster.coordinator.service.feature</id>
```

Comment:
org.wso2.carbon.cluster.coordinator.service.feature.group instead to maintain the consistency